REUSE.toml

```diff
@@ -23,3 +23,10 @@ path = [
 ]
 SPDX-FileCopyrightText = "2022 CERN"
 SPDX-License-Identifier = "CC0-1.0"
+
+[[annotations]]
+path = [
+        "cmake/.gitattributes",
+]
+SPDX-FileCopyrightText = "CERN and covfie contributors"
```

Comment:
See https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects